File: delete_files.py
import os

files = [
    r'c:\Users\abdel\Desktop\تهنئة العيد\fix_arabic.js',
    r'c:\Users\abdel\Desktop\تهنئة العيد\fix_arabic.py',
    r'c:\Users\abdel\Desktop\تهنئة العيد\tempfix.js',
    r'c:\Users\abdel\Desktop\تهنئة العيد\tempfix.py',
    r'c:\Users\abdel\Desktop\fix_arabic.js',
    r'c:\Users\abdel\Desktop\fix_arabic.py'
]

for f in files:
    try:
        if os.path.exists(f):
            os.remove(f)
            print(f'DELETED: {f}')
        else:
            print(f'NOT FOUND: {f}')
    except Exception as e:
        print(f'ERROR deleting {f}: {e}')

print('\nDeletion process completed')

File: delete_files.js
const fs = require('fs');
const path = require('path');

const baseDir = 'c:\\Users\\abdel\\Desktop\\تهنئة العيد';
const files = ['fix_arabic.js', 'fix_arabic.py', 'tempfix.js', 'tempfix.py'];

console.log('Attempting to delete files...\n');

files.forEach(file => {
  const filePath = path.join(baseDir, file);
  try {
    fs.unlinkSync(filePath);
    console.log(`✓ DELETED: ${file}`);
  } catch (err) {
    if (err.code === 'ENOENT') {
      console.log(`✗ NOT FOUND: ${file}`);
    } else {
      console.log(`✗ ERROR (${file}): ${err.message}`);
    }
  }
});

console.log('\nDone.');
